apps/ledger-live-mobile/src/families/celo/RevokeFlow/VoteAmount.tsx

```diff
@@ -96,6 +98,8 @@ export default function VoteAmount({ navigation, route }: Props) {
   const unit = useAccountUnit(account);
   const error = amount.eq(0) || bridgePending ? null : getFirstStatusError(status, "errors");
   const warning = getFirstStatusError(status, "warnings");
+  const isRevoking = transaction.mode === "revoke";
```

Comment:
Do we need this as we are already in the revoke flow ? The file is in the `RevokeFlow` directory

Reply:
removed ✅